deps/Makefile

```diff
@@ -212,5 +221,6 @@ cleanall:
 	cd libssl && rm -rf openssl-1.1.1b || true
 	cd libssl && rm -rf openssl-1.1.1d || true
 	cd libconfig && rm -rf libconfig-1.7.2 || true
+	cd prometheus-cpp && make clean
```

Comment:
Follow the same convention of all other dependencies. Remove the directory of the unpacked tarball

Reply:
Done!